src/services/produto.service.js

```diff
@@ -213,3 +228,43 @@ export const getHomologacoesCODAE = () => {
       return error;
     });
 };
+
+export const getReclamacaoDeProduto = () => {
+  const url = `${API_URL}/painel-gerencial-homologacoes-produtos/filtro-por-status/codae_autorizou_reclamacao/`;
+  return retornoBase(url);
+};
+
+export const getProdutosSuspensos = () => {
+  const url = `${API_URL}/painel-gerencial-homologacoes-produtos/filtro-por-status/codae_suspendeu/`;
+  return retornoBase(url);
+};
+
+export const getCorrecaoDeProduto = () => {
+  const url = `${API_URL}/painel-gerencial-homologacoes-produtos/filtro-por-status/codae_questionado/`;
+  return retornoBase(url);
+};
+
+export const getAguardandoAnaliseReclamacao = () => {
+  const url = `${API_URL}/painel-gerencial-homologacoes-produtos/filtro-por-status/codae_pediu_analise_reclamacao/`;
+  return retornoBase(url);
+};
+
+export const getAguardandoAnaliseSensorial = () => {
+  const url = `${API_URL}/painel-gerencial-homologacoes-produtos/filtro-por-status/codae_pediu_analise_sensorial/`;
+  return retornoBase(url);
+};
+
+export const getPendenteHomologacao = () => {
+  const url = `${API_URL}/painel-gerencial-homologacoes-produtos/filtro-por-status/codae_pendente_homologacao/`;
+  return retornoBase(url);
+};
+
+export const getHomologados = () => {
+  const url = `${API_URL}/painel-gerencial-homologacoes-produtos/filtro-por-status/codae_homologado/`;
+  return retornoBase(url);
+};
+
+export const getNaoHomologados = () => {
+  const url = `${API_URL}/painel-gerencial-homologacoes-produtos/filtro-por-status/codae_nao_homologado/`;
```

Comment:
```suggestion
const BASE_PATH = ${API_URL}/painel-gerencial-homologacoes-produtos/filtro-por-status
  const url = retornoBase(`${BASE_PATH}/codae_nao_homologado/`);
```
Dá pra usar o mesmo path para as 8 funções, mas o ideal aqui é não criar mais de uma função já que todas fazem a mesma coisa. Basta passar uma constante como parâmetro que a função usa o parametro para montar o _path_: 
import {  STATUS_HOMOLOGACAO_DE_PRODUTO as STATUS }  from "constants/shared"
obterDadosPainelGerencial(STATUS.CODAE_NAO_HOMOLOGADO)

Reply:
converti pra um único endpoint:
```
export const getHomologacoesDeProdutoPorStatus = status => {
  const url = `${API_URL}/painel-gerencial-homologacoes-produtos/filtro-por-status/${status}/`;
  return retornarBase(url);
};
```
e o status eu pego dessa constante:
```
export const ENDPOINT_HOMOLOGACOES_PRODUTO_STATUS = {
  CODAE_AUTORIZOU_RECLAMACAO: "codae_autorizou_reclamacao",
  CODAE_SUSPENDEU: "codae_suspendeu",
  CODAE_QUESTIONADO: "codae_questionado",
  CODAE_PEDIU_ANALISE_RECLAMACAO: "codae_pediu_analise_reclamacao",
  CODAE_PEDIU_ANALISE_SENSORIAL: "codae_pediu_analise_sensorial",
  CODAE_PENDENTE_HOMOLOGACAO: "codae_pendente_homologacao",
  CODAE_HOMOLOGADO: "codae_homologado",
  CODAE_NAO_HOMOLOGADO: "codae_nao_homologado"
}

```